doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -153,6 +153,11 @@ HomeKit
 
 HomeKit is now removed, as announced in the :ref:`ncs_release_notes_250`.
 
+Core firmware
+=============
+
+* Added the :ref:`ipc_radio` firmware image.
```

Comment:
```suggestion
* Added the :ref:`core_fw` page.
```
This could be mentioned also in the Documentation section like this:
* Added :ref:`core_fw` documentation.

Reply:
Does it mean to add 2 entries - in Documentation and as a Core firmware?

Reply:
Yes.